php7.0-fpm/Dockerfile

```diff
@@ -8,14 +8,15 @@ RUN apt-get update && apt-get install -y \
         libcurl4-gnutls-dev \
         libicu-dev \
         zlib1g-dev \
-        npm \
         git \
+        build-essential \
     && docker-php-ext-install -j$(nproc) iconv mcrypt pdo_mysql intl opcache mbstring curl zip \
     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
     && docker-php-ext-install -j$(nproc) gd
 
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-RUN apt-get clean
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
```

Comment:
@oanhnn I think we should move this line to the top of Dockerfile.
So we can remove the next line (`RUN apt-get install -y nodejs && apt-get autoremove && apt-get clean`)  and add `nodejs` after `build-essential`.

So our image will be reduced 1 layer than before.